src/overmap.cpp

```diff
@@ -1979,9 +1990,15 @@ void overmap::draw(WINDOW *w, WINDOW *wbar, const tripoint &center,
         } else {
             mvwputch(wbar, 1, 1, otermap[ccur_ter].color, otermap[ccur_ter].sym);
             std::vector<std::string> name = foldstring(otermap[ccur_ter].name, 25);
-            for (size_t i = 0; i < name.size(); i++) {
+            size_t i;
+            for (i = 0; i < name.size(); i++) {
                 mvwprintz(wbar, i + 1, 3, otermap[ccur_ter].color, "%s", name[i].c_str());
             }
+
+            std::vector<std::string> closeup_name = foldstring(overmap_buffer.closeup_name(cursx,cursy,z), 25);
+            for (size_t j = 0; j < closeup_name.size(); j++) {
+                mvwprintz(wbar, j + i + 1, 1, otermap[ccur_ter].color, "- %s", closeup_name[i].c_str());
+            }
```

Comment:
At various points in my debugging this has printed garbage, or nothing, or the desired string. I couldn't get consistent behavior as I investigated.